docs/src/modules/java/pages/ai-coding-assistant.adoc

```diff
@@ -2,7 +2,7 @@
 
 include::ROOT:partial$include.adoc[]
 
-AI coding assistants can increase your productivity when developing Akka services. This guide will give you some practical hints of how to setup Akka knowledge and how to prompt the AI assistant. We are using https://www.qodo.ai[Qodo, window="new"] and https://www.cursor.com[Cursor, window="new"] as examples of such coding assistants, but the techniques are applicable for other tools as well.
+AI coding assistants can increase your productivity when developing Akka services. This guide will give you some practical hints of how to setup Akka knowledge and how to prompt the AI assistant. We are using https://www.qodo.ai[Qodo, window="new"], https://www.cursor.com[Cursor, window="new"] and IntelliJ IDEA as examples of such coding assistants, but the techniques are applicable for other tools as well.
```

Comment:
As said in the end of this sentence, it's most important to show different ways that the assistants may use. Therefore good to also include IntelliJ since it's a slightly different approach.